epochStart/metachain/economics.go

```diff
@@ -406,6 +407,15 @@ func (e *economics) startNoncePerShardFromEpochStart(epoch uint32) (map[uint32]u
 	return mapShardIdNonce, previousEpochStartMeta, nil
 }
 
+func (e *economics) maxPossibleNotarizedBlocks(currentRound uint64, prev *block.MetaBlock) uint64 {
+	maxBlocks := uint64(0)
+	for _, shardData := range prev.EpochStart.LastFinalizedHeaders {
+		maxBlocks += currentRound - shardData.Round
+	}
+
+	return maxBlocks
```

Comment:
max blocks does not contain metachain rounds/blocks

Reply:
done, good find